commit d13cb1516cb5ae4cb8322ed630e1d4e1f584fd77
Author: Jens Bissinger <whiterabbit.init@googlemail.com>
Date:   Thu Aug 23 14:11:27 2018 +0200

    DOC: Fix spelling error in configuration doc
    
    Fix spelling error in logging section of configuration doc.
    
    (cherry picked from commit 15c64ff4fb9f1f64b31306ac53b38fc4d5fb1538)
    Signed-off-by: Willy Tarreau <w@1wt.eu>

diff --git a/doc/configuration.txt b/doc/configuration.txt
index 43e28785..0dd212ad 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -16089,7 +16089,7 @@ Please refer to the table below for currently defined variables :
   |   | %t   | date_time      (with millisecond resolution)  | date        |
   | H | %tr  | date_time of HTTP request                     | date        |
   | H | %trg | gmt_date_time of start of HTTP request        | date        |
-  | H | %trl | locla_date_time of start of HTTP request      | date        |
+  | H | %trl | local_date_time of start of HTTP request      | date        |
   |   | %ts  | termination_state                             | string      |
   | H | %tsc | termination_state with cookie status          | string      |
   +---+------+-----------------------------------------------+-------------+
